x-pack/legacy/plugins/siem/public/pages/network/network.tsx

```diff
@@ -4,73 +4,60 @@
 import { manageQuery } from '../../components/page/manage_query';
-import { KpiNetworkComponent, NetworkTopNFlowTable } from '../../components/page/network';
-import { NetworkDnsTable } from '../../components/page/network/network_dns_table';
-import { GlobalTime } from '../../containers/global_time';
+import { KpiNetworkComponent } from '../../components/page/network';
 import { KpiNetworkQuery } from '../../containers/kpi_network';
 import { NetworkFilter } from '../../containers/network';
-import { NetworkDnsQuery } from '../../containers/network_dns';
-import { NetworkTopNFlowQuery } from '../../containers/network_top_n_flow';
+import { SiemNavigation } from '../../components/navigation';
+import { hasMlUserPermissions } from '../../components/ml/permissions/has_ml_user_permissions';
+import { MlCapabilitiesContext } from '../../components/ml/permissions/ml_capabilities_provider';
+
 import { indicesExistOrDataTemporarilyUnavailable, WithSource } from '../../containers/source';
-import { FlowTargetNew, LastEventIndexKey } from '../../graphql/types';
+import { LastEventIndexKey } from '../../graphql/types';
 import { networkModel, networkSelectors, State } from '../../store';
 import { setAbsoluteRangeDatePicker as dispatchSetAbsoluteRangeDatePicker } from '../../store/inputs/actions';
-import { InputsModelId } from '../../store/inputs/constants';
 import { SpyRoute } from '../../utils/route/spy_routes';
 import { NetworkKql } from './kql';
 import { NetworkEmptyPage } from './network_empty_page';
 import * as i18n from './translations';
 
-const NetworkTopNFlowTableManage = manageQuery(NetworkTopNFlowTable);
-const NetworkDnsTableManage = manageQuery(NetworkDnsTable);
-const KpiNetworkComponentManage = manageQuery(KpiNetworkComponent);
-
-const ConditionalFlexGroup = styled(EuiFlexGroup)`
-  ${({ theme }) => css`
-    @media only screen and (min-width: 1441px) {
-      flex-direction: row;
-    }
-  `}
-`;
-ConditionalFlexGroup.displayName = 'ConditionalFlexGroup';
+import { navTabsNetwork } from './nav_tabs';
+import { NetworkComponentProps } from './types';
 
-interface NetworkComponentReduxProps {
-  filterQuery: string;
-  queryExpression: string;
-  setAbsoluteRangeDatePicker: ActionCreator<{
-    id: InputsModelId;
-    from: number;
-    to: number;
-  }>;
-}
+import { NetworkTabs } from './network_tabs';
 
-type NetworkComponentProps = NetworkComponentReduxProps & Partial<RouteComponentProps<{}>>;
+const KpiNetworkComponentManage = manageQuery(KpiNetworkComponent);
+const sourceId = 'default';
 
 const NetworkComponent = React.memo<NetworkComponentProps>(
-  ({ filterQuery, queryExpression, setAbsoluteRangeDatePicker }) => (
-    <>
-      <WithSource sourceId="default">
-        {({ indicesExist, indexPattern }) =>
-          indicesExistOrDataTemporarilyUnavailable(indicesExist) ? (
-            <StickyContainer>
-              <GlobalTime>
```

Comment:
moved `GlobalTime` here https://github.com/elastic/kibana/pull/47556/files#diff-837a0ad8db92736e389aba8d96e8d13cR27 to match structure from `hosts`